.travis.yml

```diff
@@ -16,7 +21,12 @@ before_install:
 - sudo make install-python-deps
 - make pull-linters
 install:
-- "./test/scripts/install_minikube.sh"
+# Installing Minikube. Taken from https://kinvolk.io/blog/2017/10/running-kubernetes-on-travis-ci-with-minikube/
+- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$KUBERNETES_VERSION/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+- curl -Lo minikube https://storage.googleapis.com/minikube/releases/$MINIKUBE_VERSION/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+- sudo minikube start --vm-driver=none --kubernetes-version=$KUBERNETES_VERSION
+- minikube update-context
+- JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl get nodes -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1; done
```

Comment:
This is so much cleaner and easier to understand than the script was.

I don't think any of our docs have anything about running make install, but we might want to rename this to travis-install. Running this on your laptop would be very bad.

Reply:
Just ignore me. This is the travis config file not the make file.  It's been a morning.